config/common-config.yaml

```
@@ -77,6 +77,73 @@ clusterApi:
   - sc
   - wc
 
+crossplane:
+  resources:
+    requests:
+      cpu: 10m
+      memory: 300Mi
```

Make this closer to the limit to not fool the scheduler too much

```suggestion
      cpu: 150m
      memory: 400Mi
```